File: Models/ViewModels/RegisterModel.cs
using System;

namespace XYZToDo.Models.ViewModels
{
    public class RegisterModel
    {
        public string Username { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Avatar { get; set; }
        public decimal? CellCountry { get; set; }
        public decimal? Cell { get; set; }
    }

}

File: src/app/model/register.model.ts
import { Injectable } from '@angular/core';

Injectable()
export class RegisterModel {
    constructor() { }

    public Name: string
    public Surname: string
    public Username: string
    public Email: string
    public Password: string
    public Avatar: string
    public CellCountry?: number=90; //Turkey by default
    public Cell?: number
}
